database.sql

```diff
@@ -1867,6 +1867,16 @@ CREATE TABLE IF NOT EXISTS `subscription` (
 	FOREIGN KEY (`uid`) REFERENCES `user` (`uid`) ON UPDATE RESTRICT ON DELETE CASCADE
 ) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Push Subscription for the API';
 
+--
+-- TABLE test-full-text-search
+--
+CREATE TABLE IF NOT EXISTS `test-full-text-search` (
```

Comment:
This feels like a debug name. What did you mean with `test`? There might be a better term that doesn't evoke either unit tests or debug code.

Reply:
What about "check" instead?

Reply:
"check" sounds better.